helpers/constans.js

```diff
@@ -24,6 +24,9 @@ export function dayHelper(time) {
   if (time.isFriday()) {
     return REASONS_TO_NOT_DEPLOY
   }
+  if (time.isThursdayAfternoon()) {
+    return REASONS_FOR_THURSDAY_AFTERNOON
```

Comment:
You forget to import `REASONS_FOR_THURSDAY_AFTERNOON` here.

Reply:
Ooops, thanks! Fixed that.